lib/cluster.js

```diff
@@ -307,17 +307,19 @@ function masterInit() {
     workerEnv = util._extend(workerEnv, env);
     workerEnv.NODE_UNIQUE_ID = '' + id;
 
-    for (var i = 0; i < execArgv.length; i++) {
-      var match = execArgv[i].match(/^(--debug|--debug-brk)(=\d+)?$/);
+    if (1024 < debugPort < 65535) {
```

Comment:
@sam-github I don't believe so; these numbers are hardcoded in node.cc:

https://github.com/joyent/node/blob/4d9c81b7e2522c5d5d9d35058cbb0bce1228d360/src/node.cc#L2903